CMake/Dependencies/libkvspic-CMakeLists.txt

```diff
@@ -7,7 +7,7 @@ include(ExternalProject)
 # clone repo only
 ExternalProject_Add(libkvspic-download
 	GIT_REPOSITORY    https://github.com/awslabs/amazon-kinesis-video-streams-pic.git
-	GIT_TAG           98cc13510eda7da08eb91cb6904c8f980aa110af
+	GIT_TAG           add-stream-stateMachine-iterator
    	SOURCE_DIR        "${CMAKE_CURRENT_BINARY_DIR}/kvspic-src"
```

Comment:
Is the commit that you're trying to use not in PIC's develop yet?

Reply:
It is now, this PR was opened to auto run the CI tests, forgot to close